build_depends.repos

```diff
@@ -0,0 +1 @@
+repositories:
```

Comment:
Removed this dependency in https://github.com/autowarefoundation/autoware_adapi_msgs/pull/7.